ibm/provider/provider.go

```diff
@@ -1006,9 +1039,40 @@ func Provider() *schema.Provider {
 			"ibm_en_subscription_ios":     eventnotification.ResourceIBMEnFCMSubscription(),
 			"ibm_en_subscription_chrome":  eventnotification.ResourceIBMEnFCMSubscription(),
 			"ibm_en_subscription_firefox": eventnotification.ResourceIBMEnFCMSubscription(),
-			"ibm_en_subscription_slack":   eventnotification.ResourceIBMEnSlackSubscription(),
```

Comment:
Don't remove the other serivce resources/datasources

Reply:
They're not removed. They're further down in the file, I think the git merge put them there. I'll move them to the correct location.